core/sql/sqlcomp/CmpSeabaseDDLmd.h

```diff
@@ -115,7 +115,8 @@ static const QString seabaseColumnsDDL[] =
   {"   hbase_col_qualifier varchar(40) character set iso88591 not null not serialized, "},
   {"   direction char(2) character set iso88591 not null not serialized, "},
   {"   is_optional char(2) character set iso88591 not null not serialized, "},
-  {"   flags largeint not null not serialized "},
+  {"   flags largeint not null not serialized, "},
+  {"   comment VARCHAR(1000) CHARACTER SET UTF8 NOT NULL  NOT SERIALIZED "},
```

Comment:
Instead of requiring a metadata change, you may want to add a new text_type to the text table and store comments there.  We do this for other features.  When we decide to do an upgrade, we will remove the text entries from TEXT table and load them into the new metadata column COMMENT.

Reply:
Need more time to do this TEXT table modification.